specs/waku/v2/waku-store-pagination.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Waku
+version: 2.0.0-beta1
+status: Draft
+authors: 
+---
+
+# Table of Contents
+
+- [Abstract](#abstract)
+- [Wire Specification](#wire-specification)
+  * [Protobuf](#protobuf)
+
+# Abstract
+
+Pagination is an optional service enabled on top of the `WakuStore` protocol. By using pagination, a requester can set the number of waku messages to return and the offset to start from. The queried node MUST then use this information to return messages in the queried range. The details of the pagination are explained next.
```

Comment:
Could also be phrased as something like:

Nodes supporting store protocol MAY (or perhaps SHOULD) support Pagination. When using pagination, foo bar baz...

Reply:
Sure, will modify it